plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -98,6 +114,7 @@ export interface RouterOptions {
   additionalTemplateFilters?: Record<string, TemplateFilter>;
   additionalTemplateGlobals?: Record<string, TemplateGlobal>;
   permissionApi?: PermissionEvaluator;
+  customPermissionRules?: ScaffolderPermissionRuleInput[];
```

Comment:
I chose to name the optional param `customPermissionRules`, but am curious if others have thoughts on this. The catalog has a builder and calls the function `addPermissionRules`; we could try to be more consistent with the catalog and change this name to `permissionRules`, but then would need to change the variable name on line 289. Up for suggestions!